builtin/mainmenu/init.lua

```diff
@@ -85,7 +85,15 @@ local function init_globals()
 		end,
 		-- Filter function
 		function(element, gameid)
-			return element.gameid == gameid
+			if element.gameid == gameid then
+				return true
+			end
+			for _, alias in ipairs(pkgmgr.find_by_gameid(gameid).aliases) do
+				if pkgmgr.normalize_game_id(element.gameid) == pkgmgr.normalize_game_id(alias) then
```

Comment:
aren't these already normalized?
also it would make more sense for `aliases` to be a set (`{foo=true, bar=true}`) so you don't need to iterate

Reply:
Not sure if these was normalized, I think it was a previous review that requested that I normalize those.

Reply:
Also it's not a set because it's used with table.concat() where it is made (which is your next review point, the one about setting conf).

Reply:
> Not sure if these are normalized,

Yes they are, look how `getSubgameSpec` normalizes them.

Reply:
Refactored, removed all calls that weren't required.

Verified that all cases still work.